src/components-examples/angular/autocomplete/autocomplete-display-with/autocomplete-display-with-example.ts

```diff
@@ -17,11 +17,11 @@ interface ExampleOption {
   templateUrl: 'autocomplete-display-with-example.html',
 })
 export class AutocompleteDisplayWithExample {
-  myControl = new UntypedFormControl('');
+  myControl = new FormControl<string | ExampleOption>('');
 
   filteredOptions: Observable<ExampleOption[]> = this.myControl.valueChanges.pipe(
     startWith(''),
-    map((newValue) =>
+    map((newValue: any) =>
```

Comment:
would this work without any?

Reply:
Yes, I've updated it to make it more explizit.